pubtools/_pulp/tasks/push/command.py

```diff
@@ -135,7 +138,8 @@ def add_phase(klass, **kwargs):
 
         if self.args.pre_push:
             # If we are in pre-push mode then we do not go any further, we just wait
-            # for all previous steps, then log a message and exit.
+            # for all previous steps, do post actions for push items, then log a message and exit.
+            add_phase(PostPushActions)
```

Comment:
I'm not sure this should be invoked for the pre-push case.  Given that we are saying the hook is invoked when an item is pushed, and pre-push doesn't really push them (make things available on the CDN), it doesn't seem to fit.

If it is necessary for your use-case - how do you plan to differentiate pushed vs non-pushed items?  If you don't plan to use it in the pre-push case I'd rather not add it.

Reply:
Ok, I added this only to support pre-push type of task. It's not needed for my use case